content/dstu2.md

```diff
@@ -37,7 +37,202 @@ Status: 200 OK
+{
+  "resourceType": "Conformance",
+  "text": {
+    "status": "generated",
+    "div": "\u003Cdiv\u003EGenerated Conformance Statement\u003C/div\u003E"
```

Comment:
I would expect it to show just like it does on the conformance page - with the angle brackets (escape with &lt so that it's not treated as html). Both spots this was changed.